bundles/org.openhab.ui.basic/web-src/_layout.scss

```diff
@@ -1009,13 +1009,47 @@ input::-webkit-calendar-picker-indicator {
 	}
 }
 
-// Switch question mark when UNDEF or NULL
+// Question mark when UNDEF or NULL
 .unknown-state .mdl-switch__thumb::after {
 	content: "?";
-	color: var(--switch-unknown-color);
+	color: var(--state-unknown-color);
 	font-size: 14px;
 	position: absolute;
 	top: 50%;
 	left: 50%;
 	transform: translate(-50%, -50%);
 }
+.mdl-slider__container.unknown-state {
+	position: relative;
+	overflow: visible !important;
+}
+.mdl-slider__container.unknown-state::after {
+	content: "?";
+	color: var(--state-unknown-color);
+	font-size: 14px;
```

Comment:
Question: no need to set font-weight to bold like you do for color picker and color temperature picker ?

Same question for .unknown-state .mdl-switch__thumb::after

Reply:
As you can see in the screenshots, I didn't need bold here. I used the font weight from the switch and only added bold to color picker and color temperature picker because it did look much less heavy then for the switch and slider without.